kpatch-build/lookup.c

```diff
@@ -135,12 +147,11 @@ static int locals_match(struct lookup_table *table, int idx,
 				break;
 			if (sym->bind != STB_LOCAL)
 				continue;
-			if (sym->type != STT_FUNC && sym->type != STT_OBJECT)
-				continue;
```

Comment:
If my above suggestion to check for STT_{FUNC,OBJECT} is taken, this hunk probably doesn't need removed.